### ISSUE TITLE
Origins dissapear

### ISSUE THREAD

**Issue by Grimsnarl** [EdwinMindcraft/origins-architectury#24]
HI! I use this on a server of mine, When logging into the server for the first time with the mod there was no prompt to choose an origin. I had to do it manually. And Ocassionally, quite often actually, the origin abilities disappear completely. However we are still affected by the environment. So for example I'm an enderian, hitting g does not allow me to teleport however water and rain still injurs me. It happens at random intervals and sometimes it turns back on. It's random and some people will have abilities while other people do not. When pressing O as well it does not indicate that the player even has an origin.

We have not used any datapacks on the mod yet but we are planning to.
I'm attaching the list of mods we use on the server just so you can have them.
![image](https://user-images.githubusercontent.com/84813516/119692005-0a3a8800-be19-11eb-9067-2c6f538b1110.png)

**Comment by EdwinMindcraft** [comment]
Please try updating origins to the latest version on curseforge as it may solve some of your problems. The version you are currently using is broken.

**Comment by Grimsnarl** [comment]
I just tested it and it is still an issue. For me and other players. Upon closing the Minecraft launcher reloading back into the server, the origin will disappear. pressing O presents this screen
![image](https://user-images.githubusercontent.com/84813516/120703099-e0213f80-c482-11eb-9eca-e4543f3cf378.png)

mods
![image](https://user-images.githubusercontent.com/84813516/120703163-f4653c80-c482-11eb-80d9-2074eb212be0.png)

**Comment by EdwinMindcraft** [comment]
Version 0.7.3.2 will be up shortly with a possible fix. I have no clue if it will actually work because my setup doesn't actually show networking issues.
You'll need to update both the client and the server to be able to connect with this release.

**Comment by Grimsnarl** [comment]
I'll try testing it out tomorrow! Or I can get my server to maybe do it today but I will update you! thank you for your responses and your help! I really appreciate it

**Comment by Grimsnarl** [comment]
> Version 0.7.3.2 will be up shortly with a possible fix. I have no clue if it will actually work because my setup doesn't actually show networking issues.
> You'll need to update both the client and the server to be able to connect with this release.

HI! So I tried the update out. Before if you logged onto the server you would randomly have your origin and it would randomly disappear between dimensions. So Going to the nether or end might take it away or give it to you. Trying this update, logging onto the server gives no origin but going into the nether or end gives you the origin you last applied to yourself. G ability works and O brings up your origin screen. If you relog while in the end or nether the origin will disappear upon relog but reappear upon entering the overworld.  I tried relogging on the server around 20 times and the origin no longer randomly disappears but is just not there altogether until you go into a portal of some sort. This was consistent throughout every trial. HOPE THIS HELPS

**Comment by EdwinMindcraft** [comment]
When you have time, I've released 0.7.3.3 with another tentative fix. Same as before, not sure it'll work, but if you can look out for errors: 
* An error occurred while synchronizing origins for ...
* Couldn't find component for entity ...
If you get either in your client's log it probably won't work.

**Comment by Grimsnarl** [comment]
Tested the update, neither error appeared in the clients log, Runs exactly the same as in the last update. origin still disappears with relog are reappears when going into the nether or end.

**Comment by EdwinMindcraft** [comment]
The latest release should sync origins every 6.4s so this is technically "fixed", but not the right way.

**Comment by Grimsnarl** [comment]
Yeah! it works! That's awesome! Do you think it will get a proper fix? without the recheck?

Thank you so much for all your work! I really appreciate you working with me to get this sorted

**Comment by EdwinMindcraft** [comment]
I hope I'll be able to understand why packets sent on login aren't received, but I'm not really familiar with the libraries used, so no promises. 
I'm keeping this issue open as a reminder that this is something I need to do, since the "fix" is a workaround more than anything else.
